rwandaemr-imb-butaro/pom.xml

```diff
@@ -138,29 +138,26 @@
                         <goals>
                             <goal>install-node-and-npm</goal>
                         </goals>
-                        <phase>compile</phase>
                         <configuration>
-                            <nodeVersion>v16.15.1</nodeVersion>
+                            <nodeVersion>v18.13.0</nodeVersion>
                         </configuration>
                     </execution>
                     <execution>
-                        <id>Build frontend application</id>
+                        <id>Assemble frontend modules</id>
                         <goals>
                             <goal>npx</goal>
                         </goals>
-                        <phase>compile</phase>
                         <configuration>
-                            <arguments>openmrs@next build --build-config target/spa-build-config.json --target target/distro/web/frontend</arguments>
+                            <arguments>--legacy-peer-deps openmrs@next assemble --mode config --config spa-build-config.json --target target/dist</arguments>
                         </configuration>
                     </execution>
                     <execution>
-                        <id>Assemble frontend modules</id>
+                        <id>Build frontend application</id>
                         <goals>
                             <goal>npx</goal>
                         </goals>
-                        <phase>compile</phase>
                         <configuration>
-                            <arguments>openmrs@next assemble --mode config --config target/spa-build-config.json --target target/distro/web/frontend</arguments>
+                            <arguments>--legacy-peer-deps openmrs@next build --build-config spa-build-config.json --target target/dist</arguments>
```

Comment:
All of these changes are per updates to the 3.x build process, and copied from our other distribution